docs/source/contributing/build/windows.rst

```diff
@@ -8,19 +8,23 @@ Dependencies
 Qt
 ----------------------------------------------------------
 Download the `Qt online installer`_. Run it and select the latest Qt version
-for installation (e.g. ``Qt 5.10.1``). Select "MinGW 5.3.0" in section ``Tools``.
+for installation (e.g. ``Qt 5.15.0`` with ``MinGW 8.1.0 32-bit`` in it). Then select ``MinGW 8.1.0 32-bit`` in section ``Developer and Designer Tools``. The first option installs QT Creator, qmake etc. The second option installs gcc, gdb etc. (If you want you can also choose ``QT Creator CDB Debugger Support`` and ``Debugging Tools for Windows`` in section ``Developer and Designer Tools``.)
 
 Other Libraries
 ----------------------------------------------------------
  1. Download precompiled `MediaInfo <https://mediaarea.net/de/MediaInfo/Download/Windows>`_ (DLL)
  2. Download `MediaInfoLib <https://github.com/MediaArea/MediaInfoLib>`_. |br|
     Copy the folder ``Source/MediaInfoDLL`` to ``path/to/MediaElch/MediaInfoDLL``
  3. Download `ZenLib <https://github.com/MediaArea/ZenLib>`_. |br|
-    Copy the folder ``Source/ZenLib`` to ``path/to/MediaElch/ZenLib``
+    Copy the folder ``Source/ZenLib`` to ``path/to/MediaElch/ZenLib`` |br|
+ 4. Download and run `MSYS2 installer <https://www.msys2.org/>`_. |br|
+    Install ``mingw-w64-i686-quazip`` using included `pacman <https://www.msys2.org/>`_ utility |br|
+    Copy the folder ``msys64/mingw32/include/quazip5`` to ``path/to/MediaElch/quazip5`` |br|
```

Comment:
Interesting. That should not be necessary. MediaElch compiles it's own quazip. Did you also initialize MediaElch's git submodules? :-)

Reply:
I tried to put contents of quazip-master (https://github.com/stachenov/quazip) into path/to/MediaElch/third_party/quazip. I was also digging around with zlib.h and zconf.h. With this setup it did not compile.

Reply:
Forgot to mention zconf.h and zlib.h files (6e081d2).

Reply:
Ah, ok. zlib may indeed produce issues...

PS: You can initialize git submodule using `git submodule init`

Reply:
Oh, thanks. I did not know `git submodule init`. And after that `git submodule update` cloned quazip into path/to/MediaElch/third_party/quazip. Unfortunately, it did not compile also complaining about undefined references from zlib.h. Copying zlib.h and zconf.h into path/to/MediaElch/third_party/quazip/quazip did not help too.